### DISCUSSION TITLE
Version numbering stable?

### DISCUSSION THREAD

**Discussion by timsneath** [microsoft/win32metadata#741] [Q&A] [answered]
Is the numbering model stable now for Windows.Win32? I'm thinking of updating my Dart winmd package versioning to match the winmd version it consumes, but I'd like to know that we're going to continue with the same model first...

**Comment by sotteson1** [comment, marked as answer]
In the future we will continue to update the major number for breaking changes (like moving items in or out of namespaces). If that's what you mean, yes, we will stay with that model in the future.

**Comment by timsneath** [comment]
Yeah, that's what I was looking for -- you're not going to (for example) change it to match a Windows build number or something else. Thanks!

**Reply by sotteson1** [reply]
No, we won't do that.
